src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/TemplateContentLoader.cs

```diff
@@ -22,14 +22,14 @@ private static T RequireService<T>(IServiceProvider provider) where T : class
             T result = provider.GetService(typeof(T)) as T;
             if (result == null)
             {
-                throw new InvalidOperationException(SR.Format(SR.DeferringLoaderNoContext,typeof(TemplateContentLoader).Name, typeof(T).Name));
+                throw new InvalidOperationException(SR.Format(SR.DeferringLoaderNoContext,nameof(TemplateContentLoader), typeof(T).Name));
```

Comment:
Nit, since you're changing that line:
```suggestion
                throw new InvalidOperationException(SR.Format(SR.DeferringLoaderNoContext, nameof(TemplateContentLoader), typeof(T).Name));
```

Reply:
Thanks!